rxandroidble/src/main/java/com/polidea/rxandroidble/internal/connection/RxBleConnectionImpl.java

```diff
@@ -528,4 +533,32 @@ public BluetoothGattCharacteristic call(byte[] bytes) {
     public Observable<Integer> readRssi() {
         return rxBleRadio.queue(new RxBleRadioOperationReadRssi(gattCallback, bluetoothGatt, timeoutScheduler));
     }
+
+    @Override
+    public <T> Observable<T> queue(final RxBleRadioOperationCustom<T> operation) {
+        return rxBleRadio.queue(new RxBleRadioOperation<T>() {
```

Comment:
Is there anything in terms of locking the radio permanently we should add to the JavaDoc then? Possibly if this observable wouldn't emit, we will get stuck. This is not so obvious for someone reading the documentation.